runtime/src/iree/hal/string_util_test.cc

```diff
@@ -14,6 +14,7 @@
 #include "iree/base/api.h"
 #include "iree/base/internal/span.h"
 #include "iree/hal/api.h"
+#include "iree/hal/buffer_view.h"
```

Comment:
don't include - iree/hal/api.h has it

(if you have your editor doing this, turn it off)